apps/admin-gui/src/app/shared/components/entityless-attribute-keys-list/entityless-attribute-keys-list.component.html

```diff
@@ -0,0 +1,101 @@
+    (click)="onAdd()">
+    {{'SHARED.COMPONENTS.ENTITYLESS_ATTRIBUTES_LIST.ADD' | translate}}
+  </button>
+  <button
+    mat-flat-button
+    class="ml-2"
+    color="warn"
+    (click)="onRemove()">
+    {{'SHARED.COMPONENTS.ENTITYLESS_ATTRIBUTES_LIST.REMOVE' | translate}}
+  </button>
+</div>
+
+<div mat-dialog-content class="mt-3">
+  <div *ngIf="!(records.length === 0)" class="card mt-3">
+    <div class="card-body">
+      <table mat-table [dataSource]="dataSource" matSort matSortActive="key" matSortDirection="asc"
+             matSortDisableClear
+             class="w-100">
+
+        <ng-container matColumnDef="select">
+          <th mat-header-cell *matHeaderCellDef>
+            <mat-checkbox color="primary"
+                          (change)="$event ? masterToggle() : null"
+                          [checked]="selection.hasValue() && isAllSelected()"
+                          [indeterminate]="selection.hasValue() && !isAllSelected()"
+                          [aria-label]="checkboxLabel()">
+            </mat-checkbox>
+          </th>
+          <td mat-cell *matCellDef="let record" class="static-column-size">
+            <mat-checkbox color="primary"
+                          (click)="$event.stopPropagation()"
+                          (change)="$event ? selection.toggle(record) : null"
+                          [checked]="selection.isSelected(record)"
+                          [aria-label]="checkboxLabel(record)">
+            </mat-checkbox>
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="key">
+          <th mat-header-cell *matHeaderCellDef
+              mat-sort-header>
+            {{'SHARED.COMPONENTS.ENTITYLESS_ATTRIBUTES_LIST.KEY' | translate}}
+          </th>
+          <td mat-cell *matCellDef="let record">
+            <label>
+              <input class="mt-2" matInput placeholder="New key" (ngModelChange)="this.onValueChange(record)"
+                     [(ngModel)]="record[0]">
+            </label>
+          </td>
+        </ng-container>
+        <ng-container matColumnDef="value">
+          <th mat-header-cell *matHeaderCellDef
+              mat-sort-header>
+            {{'SHARED.COMPONENTS.ENTITYLESS_ATTRIBUTES_LIST.VALUE' | translate}}
+          </th>
+          <td mat-cell *matCellDef="let record">
+            <app-attribute-value
+              [attribute]="record[1]"
+              (sendEventToParent2)="onValueChange(record)">
+            </app-attribute-value>
+          </td>
+        </ng-container>
+
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr
+          mat-row
+          *matRowDef="let record; columns: displayedColumns;"
+          class="dark-hover-list-item">
+        </tr>
+      </table>
+      <mat-paginator [length]="this.records.length" [pageSize]="5" [pageSizeOptions]="[1,5,10,25]"></mat-paginator>
+    </div>
+  </div>
+
+  <div *ngIf="records.length === 0" class="perun-alert warn-alert mt-3">
```

Comment:
To use a perun alert, user ours `alert` component. It is located in `libs/ui/alerts/`

Reply:
And maybe use `info` instead of `warning`

Reply:
Fixed